interactives/high-score-boxes/js/findHighScore.js

```diff
@@ -0,0 +1,180 @@
+//onload make array of 5 random ints
+this.randomInts = [];
+this.largest = 0;
+this.numberOfBoxes = 15
+this.boxes = []
+this.gameStarted = false;
+this.secondsTaken = 0;
+this.timerVar;
+
+//this.arrayForMargins = ["top", "bottom", "right", "left"]
+
+$(document).ready(function(){
+	generateRandomNumbers();
+	createBoxObjects();
+	createBoxElements();
+
+	$("#restartButton").click(function() {
+		$("#box_holder_div").empty();
+		secondsTaken = 0;
+		gameStarted = false;
+
+		if (typeof timerVar !== 'undefined') {
+			clearTimeout(timerVar);
+		}	
+		
+		generateRandomNumbers();
+		createBoxObjects();
+		createBoxElements();
+		$("#restartButton").hide();
+		$("#completionMessage").html("");
+		$("#answerTextForm").val('');
+
+	})
+})
+
+$(document).on('click','.box', function() {
+	$(".box").stop(true, true);
+	$(".box").show();
+	for (var i = 0; i < (boxes.length); i++) { 
+		if (document.getElementById('box' + i) ==  event.target) {
+			$(this).fadeOut(1000);
+			$(this).fadeIn(1000);
+			boxes[i].revealed_times += 1;
+		}
+	}
+	//to start timer only if it hasn't started yet
+	if (!gameStarted) {
+		for (var i = boxes.length - 1; i >= 0; i--) {
```

Comment:
Can you tell me what lines 48 and 49 are for? They seem like they are not needed.